lib/usr/app/jcfg/jcfg_lport_group.c

```diff
@@ -247,6 +247,8 @@ setup_lport(jcfg_data_t *data, jcfg_lport_group_t *lpg, const char *netdev, uint
     lport->umem         = lpg->umem;
     lport->busy_timeout = lpg->busy_timeout;
     lport->busy_budget  = lpg->busy_budget;
+    lport->idle_timeout = lpg->idle_timeout;
+    lport->intr_timeout = lpg->intr_timeout;
```

Comment:
These values need to be parsed in https://github.com/CloudNativeDataPlane/cndp/blob/main/lib/usr/app/jcfg/jcfg_lport_group.c#L703

Reply:
not required anymore